src/search.cpp

```diff
@@ -1003,7 +1004,7 @@ namespace {
       else if (   PvNode
                && pos.rule50_count() > 18
                && depth < 3
-               && ++thisThread->shuffleExts < thisThread->nodes.load(std::memory_order_relaxed) / 4)  // To avoid too many extensions
+               && 4 * thisThread->previousNodes < thisThread->nodes.load(std::memory_order_relaxed))  // To avoid too many extensions
```

Comment:
I would suggest to reverse the order of operands in this condition. The following reads more naturally to me:
```
thisThread->nodes.load(std::memory_order_relaxed) > 4 * thisThread->previousNodes
```
("Is the current number of nodes more than 4 times the previous number?")

Reply:
thanks for the review... the first version of the patch was incorrect ('<' vs. '>')

Reply:
Oh, didn't spot that :) But I'd still prefer the reverse order - the usual convention is to have the value that is changing (`thisThread->nodes.load(std::memory_order_relaxed)`) on the left and the bound (`4 * thisThread->previousNodes`) on the right. I.e. you would usually write `if (i < len)` rather than `if (len > i)`. The latter would feel a bit weird.